### DISCUSSION TITLE
Featured Images in a Post List

### DISCUSSION THREAD

**Discussion by kreincke** [bootscore/bootscore#352] [Q&A]
Dear bS-Team

recently I read that an editor who uses wordpress may expect that the picture he has defined as featured image of his post / page becomes visible (= is used) at three positions: on top of the post / page int he page view, in a smaller version beside his article in the teaser list and in a very short version beside in the list of 'pure' post titles. But I also read that the theme is responsible for fulfilling these expectations.

Then I 've tested which of these expectations bootScore fulfills. My result is:

- bS successfully shows the featured images on top of the post 
- bS successfully shows the featured image left aside the post teaser. (the default start page)
- bS does not show the featured in the most recent posts widget which can be activated for the sidebar.

What is the position of bS: Shall a user of bS install another plugin which offers such a feature or is there (or should there be) an option in bS to activate this feature in bS?

Many thanks for your answers

**Comment by crftwrk** [comment]
The recent post widget is a classic WordPress widget which shows an `<ul>`, nothing more.

If you want to show posts more advanced, simply use the bS Grid plugin https://bootscore.me/documentation/bs-grid/. Shortcodes can be used in a Custom HTML widget for example in sidebar position as well. Edit plugins HTML as you want by overriding the template in your child.

Does this help?
